### PR TITLE
fix(browsers): vivaldi setup

### PR DESCRIPTION
**Pull request by leventbesli**
## Type of Change
- [ ] New feature
- [x ] Bug fix
- [ ] Documentation update
- [ ] Refactoring
- [ ] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description

Vivaldi setup section was containing Lynx install script. I replaced it with Vivaldi.

## Testing

Tested inside Fedora 40 and Debian 12 Distrobox environments. Installed successfully in both of them.

## Additional Information

First PR. Be gentle. 

## Checklist
- [ x] My code adheres to the coding and style guidelines of the project.
- [ x] I have performed a self-review of my own code.
- [ x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x ] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Review by adamperkowski — APPROVED** [review]
Hmmm...

**Review by jeevithakannan2 — CHANGES_REQUESTED** [review]
- Need to pacman and zypper in the installation
- Fix formatting
- Drop the changes to Cargo.lock

**Review by adamperkowski — APPROVED** [review]

**Review by jeevithakannan2 — CHANGES_REQUESTED** [review]
Just copy paste the the cargo.lock from the repo and push it. The cargo lock is fixed in the main branch.

File: core/tabs/applications-setup/browsers/vivaldi.sh [hunk 1]
```diff
@@ -2,22 +2,39 @@
...
+                "$ESCALATION_TOOL" "$PACKAGER" config-manager --add-repo https://repo.vivaldi.com/stable/vivaldi-fedora.repo
+                "$ESCALATION_TOOL" "$PACKAGER" install -y vivaldi-stable
+                ;;
+            zypper)
+                "$ESCALATION_TOOL" zypper ar https://repo.vivaldi.com/archive/vivaldi-suse.repo
+                "$ESCALATION_TOOL" zypper --non-interactive --gpg-auto-import-keys in vivaldi-stable
+                ;;
             pacman)
-                "$ESCALATION_TOOL" "$PACKAGER" -S --needed --noconfirm lynx
+                "$ESCALATION_TOOL" "$PACKAGER" -S --needed --noconfirm xdg-utils vivaldi
```
Comment (jeevithakannan2) [inline core/tabs/applications-setup/browsers/vivaldi.sh:26]: Why are we adding xdg-utils ? 🤔

**Review by jeevithakannan2 — CHANGES_REQUESTED** [review]

File: core/tabs/applications-setup/browsers/vivaldi.sh (hunk 1 above)
Comment (leventbesli) [reply core/tabs/applications-setup/browsers/vivaldi.sh:26]: ahh. It was from the test within headless Distrobox. It's not needed with desktop environments obviously. I will update it now. Thanks for pointing it out.

**Review by jeevithakannan2 — APPROVED** [review]

**Review by nnyyxxxx — APPROVED** [review]

**Review by nnyyxxxx — CHANGES_REQUESTED** [review]
drop the cargo lock changes or Chris will most likely decline this PR

**Comment by jeevithakannan2** [comment]
Don't do git rm cargo.lock

**Comment by jeevithakannan2** [comment]
Merge with upstream/main. It will fix the issue

File: Cargo.lock [hunk 2]
```diff
@@ -491,7 +491,7 @@ checksum = "d8adc4bb1803a324070e64a98ae98f38934d91957a99cfb3a43dcbc01bc56439"
 
 [[package]]
 name = "linutil_core"
-version = "24.9.23"
+version = "24.9.28"
```
Comment (nnyyxxxx) [inline Cargo.lock:494]:
```suggestion
version = "24.9.23"
```

File: Cargo.lock [hunk 3]
```diff
@@ -503,7 +503,7 @@ dependencies = [
 
 [[package]]
 name = "linutil_tui"
-version = "24.9.23"
+version = "24.9.28"
```
Comment (nnyyxxxx) [inline Cargo.lock:506]:
```suggestion
version = "24.9.23"
```

**Review by nnyyxxxx — CHANGES_REQUESTED** [review]
commit these changes

**Review by nnyyxxxx — APPROVED** [review]

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**